packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -674,6 +677,16 @@ export class BackgroundScript {
         }
       })
       this.log(`sendTip: sent tip to ${receiver}`, result)
+      const message: Tip = {
```

Comment:
```
const message: SendTip = {
   command: 'sendTip'
```
trivial nit: I think SendTip is a bit more in alignment with the rest of the names used

Reply:
ah, already have that, so maybe `TipSent` ?